protocol/app/upgrades/v6.0.0/upgrade_container_test.go

```diff
@@ -246,3 +260,20 @@ func postUpgradeMarketMapperRevShareChecks(node *containertest.Node, t *testing.
 		require.Equal(t, revShareDetails.Details.ExpirationTs, uint64(0))
 	}
 }
+
+func postUpgradeMarketMapState(node *containertest.Node, t *testing.T) {
+	// check that the market map state has been initialized
+	marketMap, err := containertest.Query(
+		node,
+		marketmapmoduletypes.NewQueryClient,
+		marketmapmoduletypes.QueryClient.MarketMap,
+		&marketmapmoduletypes.MarketMapRequest{},
+	)
+	require.NoError(t, err)
+
+	expectedMarkets := marketmapmoduletypes.MarketMap{
+		Markets: make(map[string]marketmapmoduletypes.Market, 0),
+	}
```

Comment:
Can you test that an existing market param is correctly converted to a market map?